packages/webview_flutter/README.md

```diff
@@ -66,5 +64,27 @@ android {
         // Required by the Flutter WebView plugin.
         minSdkVersion 19
     }
-  }
+}
+```
+
+#### Enable Material Components for Android
+
+To use Material Components when the user interacts with input elements in the WebView,
+follow the steps described in the [Getting Started guide for Android](https://material.io/develop/android/docs/getting-started). For example:
```

Comment:
How about changing this to "in the [Enabling Material Components instructions](https://flutter.dev/docs/deployment/android#enabling-material-components)" and removing everything else? That way we aren't maintaining two copies of the instructions.

Reply:
Done